src/packages/uploader/uploader.taro.tsx

```
@@ -506,6 +524,62 @@ const InternalUploader: ForwardRefRenderFunction<
+
+      if (clipboardData?.items && clipboardData.items.length) {
+        for (let i = 0; i < clipboardData.items.length; i++) {
+          const item = clipboardData.items[i]
+          if (item.kind === 'file' && item.type.startsWith('image/')) {
+            const file = item.getAsFile()
+            if (file) {
+              files.push({
+                originalFileObj: file,
+                size: file.size,
+                path: '',
+                tempFilePath: '',
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)